### PR TITLE
Invoke setup on Windows fails fixed

### LINKED ISSUE #180: *Invoke setup on Windows fails - taken via git
When running `invoke setup` on windows (powershell) it fails with the following error:

```
'eval' is not recognized as an internal or external command,
operable program or batch file.
```

We can run it via bash on WSL2 but it's linux and not windows.

This can be easily fixed by running the command without `eval` on windows [here](https://github.com/ploomber/jupysql/blob/master/tasks.py#L18)

```
    if platform.system() == "Windows":
        conda_shell_bash_hook_script = 'conda shell.bash hook '
    else:
        conda_shell_bash_hook_script = 'eval "$(conda shell.bash hook)" '
```

cc: @edublancas @idomic

### PR DESCRIPTION
**Pull request by ayushmaanshrotriya**
## Describe your changes
fixed by running the command without eval on windows
added import platform

## Issue ticket number and link
Closes #180 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added thorough [tests](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#testing) (when necessary).
- [x] I have added the right documentation in the [docstring](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#documenting-changes-and-new-features) and [changelog](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--241.org.readthedocs.build/en/241/

### CODE REVIEW

File: tempCodeRunnerFile.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import platform
```
Comment (edublancas) [inline tempCodeRunnerFile.py:1]: looks like this is a tmp file that should be removed?

File: tasks.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
+import platform
```
Comment (edublancas) [inline tasks.py:1]: please run black on this file so it passes the linting

**Review by edublancas — CHANGES_REQUESTED** [review]

File: tempCodeRunnerFile.py (hunk 1 above)
Comment (neelasha23) [reply tempCodeRunnerFile.py:1]: +1

**Comment by neelasha23** [comment]
Please ensure the CI is passing

**Comment by edublancas** [comment]
ok so the check-for-broken-links is failing  (don't know why yet) but this is unrelated to the PR so we can ignore it.

@yafimvo @neelasha23 I think you have windows, right? can either of you test it?

**Comment by neelasha23** [comment]
> @yafimvo @neelasha23 I think you have windows, right? can either of you test it?

I work on Mac

**Comment by yafimvo** [comment]
tested it on windows 11 - worked for me

```
invoke setup
```

```
invoke setup --version 3.7
```

**Review by edublancas — APPROVED** [review]

**Comment by ayushmaanshrotriya** [comment]
learning a lot by contributing into this repository.
Thanks for help

**Pull request merged**